app/javascript/gobierto_dashboards/webapp/components/Table.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div v-if="items.length">
+    <table class="dashboards-home-main--table">
+      <thead>
+        <th
+          v-for="column in columns"
+          class="dashboards-home-main--th"
+        >
+          <div>{{ column.translation }}</div>
+        </th>
+      </thead>
+      <transition-group
+        name="fade"
+        tag="tbody"
+        mode="out-in"
+      >
+        <TableRow
+          v-for="item in items"
```

Comment:
I later use `item` and `item.id` for the child component. Can I use destructuring to get both the object and one of the attributes?

Reply:
Humm, I see what happens. Let it go in the meantime, there's something I wanna explain you tomorrow (v-slot). This is a good use case.

Regarding your question, you cannot. You'd have to fetch `item` as usual. Use destructuring whenever you can use it, because it's cleaner and avoid DOM errors, but is not always available.

Reply:
Yeah, it's much better. Thanks again!